commit 3082cc5c3637d8295ecafa7e5eb4cf60b6836e4f (refs/changes/81/1681/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-03-18T00:11:46+00:00 (2 years, 7 months ago)
    
    NAPI-399 Don't run 'make test' during MG build
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/Makefile b/Makefile
index 01d9e5c..7010fb1 100644
--- a/Makefile
+++ b/Makefile
@@ -1459,7 +1459,7 @@ napi: $(NAPI_BITS) napi_image
 $(NAPI_BITS): build/sdc-napi
 	@echo "# Build napi: branch $(SDC_NAPI_BRANCH), sha $(SDC_NAPI_SHA), time `date -u +%Y%m%dT%H%M%SZ`"
 	mkdir -p $(BITS_DIR)
-	(cd build/sdc-napi && NPM_CONFIG_CACHE=$(MG_CACHE_DIR)/npm TIMESTAMP=$(TIMESTAMP) BITS_DIR=$(BITS_DIR) gmake pkg test release publish)
+	(cd build/sdc-napi && NPM_CONFIG_CACHE=$(MG_CACHE_DIR)/npm TIMESTAMP=$(TIMESTAMP) BITS_DIR=$(BITS_DIR) gmake pkg release publish)
 	@echo "# Created napi bits (time `date -u +%Y%m%dT%H%M%SZ`):"
 	@ls -l $(NAPI_BITS)
 	@echo ""
